Jenkinsfile

```diff
@@ -1,6 +1,9 @@
 pipeline {
   agent {
-    label 'qa-executors'
+    label {
+            label 'qa-executors'
+            customWorkspace "var/lib/jenkins/workspace/mobile-wiki-pr-checks_2_EXECUTOR${env.EXECUTOR_NUMBER}"
```

Comment:
why customWorkspace is needed?

Reply:
what if 2 PR checks are run at the same time?

Reply:
They run in different workspace since the Executor number is on custom workspace name

Reply:
this will prevent executor from keeping more workspaces then number of executor (that equals possible parallel runs). That said, it will occupy no more workspace than needed ;)

Reply:
Somehow I have a bad feeling about this :)
but what the hell, let's try!